README.md

```diff
@@ -98,6 +98,62 @@ resp = client.send_message(
 print(resp['requestId'])
 """
 
+Example: send UTM metadata with message object - V1
+"""
+resp = client.send_message(
+    message={
+        'template': 'my-template', 
+        'to': [
+            {
+                "email": "foo@bar.com"
+            }
+        ], 
+        'metadata': {
+            'utm': {
+                'source':'python'
+            }
+        }
+    }
+)
```

Comment:
can you remove this? v1 does not support metadata

Reply:
done